scripts/extractions/extract_sar.py

```diff
@@ -0,0 +1,400 @@
+"""Extract S1 data using OpenEO-GFMAP package."""
+import argparse
+import json
+import logging
+import os
+from datetime import datetime
+from functools import partial
+from importlib.metadata import version
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+from typing import List, Optional
+
+import geojson
+import geopandas as gpd
+import openeo
+import pandas as pd
+import pystac
+import requests
+import xarray as xr
+from openeo_gfmap import Backend, BackendContext, FetchType, TemporalContext
+from openeo_gfmap.backend import cdse_connection
+from openeo_gfmap.fetching.s1 import build_sentinel1_grd_extractor
+from openeo_gfmap.manager import _log
+from openeo_gfmap.manager.job_manager import GFMAPJobManager
+from openeo_gfmap.manager.job_splitters import (
+    _append_h3_index,
+    _load_s2_grid,
+    split_job_s2grid,
+)
+from openeo_gfmap.stac import AUXILIARY
+from shapely.geometry import Point
+
+# Logger for this current pipeline
+_pipeline_log: Optional[logging.Logger] = None
+
+
+def _setup_logger(level=logging.INFO) -> None:
```

Comment:
Why is this not part of GFMAP?

Reply:
This `_setup_logger ` function setups print in console for the logger objects in GFMAP Manager, from the extraction script and open-eo

I moved to GFMAP the setup for it's own logger, but I will leave a function here to setup the logger object of the extraction pipeline